docs/articles/deployment/container.md

```diff
@@ -106,125 +306,48 @@ We are going to use the `BYTEWAX_KEEP_CONTAINER_ALIVE` environment variable to k
 
 ```bash
 docker run --rm --name=my-dataflow \
-    -v /var/bytewax/examples:/bytewax/examples \
-    -e BYTEWAX_PYTHON_FILE_PATH=examples.pagerank:flow \
+    -v ./dataflows:/bytewax/dataflows \
```

Comment:
Now that you mention it, I remember having to use absolute paths in the past, but I copy/pasted all the commands from the guide several times before pushing, so it did work. Maybe this is fixed with buildkit? I can  put `$(pwd)` rather than `./` there just to be sure.

Reply:
Yeah, they now allow us to use relative paths in the new version. I agree with you. It sounds good to put `$(pwd)`.